android/lib/build.gradle

```diff
@@ -28,8 +28,7 @@ dependencies {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
     testImplementation 'junit:junit:4.12'
-
-    implementation 'com.walmartlabs.ern:react-native:0.59.4'
+    api ('com.walmartlabs.ern:local-container:1.0.0') {changing=true}
```

Comment:
Btw does this have to be `api`? Or can it be `implementation`?

Reply:
we need api since the app module needs to access the classes from the container.